spdx/file.py

```diff
@@ -63,7 +63,7 @@ def __init__(self, name, spdx_id=None, chk_sum=None):
         self.spdx_id = spdx_id
         self.comment = None
         self.type = None
-        self.chk_sum = chk_sum
+        self.checksums = [None]
```

Comment:
I think the intention here was to distinguish the package and file checksum by using different names `chk_sum` for file and `check_sum` for package. I agree that it is better to use `chksum` and `checksum` but I would stick to the different names for the objects since they are slightly different: checksums for packages are optional whereas checksums for files are required.

Reply:
To give my opinion here (already discussed with @meretp ):
- I find the distinction between the two variable names rather arbitrary and obscure. I would question whether it's necessary at all, and if it is, I would prefer something like `file_checksum` and `package_checksum`. That would make it clear to a reader which one is which, and also provide a hint at why the naming distinction is made in the first place
- Considering that this is probably an issue in quite a lot of places, and we are planning a major redesign in the near future, I want to be pragmatic about this. If renaming the variables everywhere is a lot of effort, let's just stick with the status quo until the redesign.